packages/node_modules/@ciscospark/react-component-utils/src/constants.js

```diff
@@ -68,3 +68,12 @@ export const SPACE_TYPE_ONE_ON_ONE = 'direct';
 export const SPACE_TYPE_GROUP = 'group';
 
 export const TAG_MUTED = 'MUTED';
+
+export const attachmentType = 'AdaptiveCard';
+export const cardReference = 'cardReference';
+export const cards = 'cards';
+
+export const DEVELOPER = 'developer';
+export const WIDGET_ADAPTIVE_CARD = 'WIDGET_ADAPTIVE_CARD';
+export const WIDGET_ADAPTIVE_CARD_OFF = 'WIDGET_ADAPTIVE_CARD_OFF';
+export const WIDGET_ADAPTIVE_CARD_ON = 'WIDGET_ADAPTIVE_CARD_ON';
```

Comment:
Since these are feature constants, please add the `FEATURES_` prefix to match the other feature constants in this file.

Reply:
@adamweeks  done with required changes.